### PR TITLE
🌐 Add catalan language

### PR DESCRIPTION
**Pull request by dragonDScript**
Added the catalan language

## Stuff to fix

- [x] Months are still in spanish. I don't know how to rename them!

### CODE REVIEW

**Comment by lostdesign** [comment]
The months might be off since you chose "code": "es-CA", guess it should be "ca-ES"

File: src/locales/es-CA.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+    "title": "Català",
+    "code": "es-CA",
```
Comment (lostdesign) [inline src/locales/es-CA.json:3]: The months might be off since you chose "code": "es-CA," I guess it should be "ca-ES," can you try that?

**Review by lostdesign — CHANGES_REQUESTED** [review]

File: src/locales/es-CA.json (hunk 1 above)
Comment (lostdesign) [reply src/locales/es-CA.json:3]: Is Catalan so much different from regular Spanish? So that it makes a difference in the app too?

Comment (dragonDScript) [reply src/locales/es-CA.json:3]: It's a completely separate language like Portuguese. Some months match the nacme and most don't.

**Comment by dragonDScript** [comment]
Fixed the ISO name

**Pull request merged**